    def raise_exception(message):
        raise Exception(f"Error message - {message}")
